scripts/install-forms-flow.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+read -p "Please enter the domain name (ex: apps.bronze.aot-technologies.com):" DOMAIN_NAME
+read -p "Please enter the namespace (ex: devin):" NAMESPACE
+
+echo "Installing forms flow to Openshift ..."
+
+helm dependency update ../charts/forms-flow-admin/
```

Comment:
Admin is not mandatory; so there should be an if condition to execute this. Ask a question like, if this premium subscription installation. Admin deploy will fail without an access token from us